code/API_definitions/quality-on-demand.yaml

```
@@ -609,6 +602,99 @@ components:
           - 5060
           - 5070
 
+    SinkCredential:
+      type: object
+      properties:
+        credentialType:
+          type: string
+          enum:
+            - PLAIN
+            - ACCESSTOKEN
+            - REFRESHTOKEN
+          description: "The type of the credential."
```

Commonalities 0.4.0 says in the Design Guidelines explicit: "Type of the credential - MUST be set to ACCESSTOKEN for now". So we should mentioned it also here:

```suggestion
          description: "The type of the credential. With the current API version the type MUST be set to `ACCESSTOKEN`"
```